_pathways/path-interactive.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Getting and Using Interactive Sessions
+last_modified_at: 2024-05-29
+primary_reviewers: atombaby
+---
+
+This pathway is a step-by-step introduction to interactive sessions in the Rhino/Gizmo compute environment.
+
+## Pre-requisites
+
+ - Basic Unix/Linux commands `cd`, `ls`, and `pwd`
+ - Connection to Campus Network (VPN or on-campus)
+ - A terminal emulator ( _Terminal_ on Mac, _Putty_, or _WSL_ on Windows)
+ - {% glossary Node %}
+ - {% glossary Shell %}
+ - {% glossary Prompt %}
+ - {% glossary Session %}
+ - {% glossary Slurm %}
+ - {% glossary HutchNet ID %}
+ - {% glossary rhino %}
+ - {% glossary gizmo %}
+
+## Steps
+
+### Get a Hutchnet ID
+
+In order to use Fred Hutch batch computing resources you must have valid Fred Hutch credentials.  Sepcicially a [HutchNet ID](/scicomputing/access_credentials/#hutchnet-id).
+
+This HutchNet ID needs to be associated with a [PI account](/scicomputing/access_credentials/#accessing-slurm-clusters) in order to submit jobs.
+
+### Connect to the Campus Network
+
+Batch computing resources require that your local computer be connected to the campus network.  This authenticaiton process happens differently when you're on vs off campus.
```

Comment:
"Authentication" is misspelled here.

Reply:
Fixed.  Sorry I had a cold when I wrote that